src/jobs/runAllJobs.ts

```diff
@@ -29,16 +39,18 @@ export default async function (
       emailClient,
     );
     const pausedAppReport = await runPauseAppsCheck(currentDate, emailClient, user);
-    const firstExpiryNotificationReport = await firstExpiryNotificationCheck(
-      currentDate,
-      emailClient,
-    );
-    const secondExpiryNotificationReport = await secondExpiryNotificationCheck(
-      currentDate,
-      emailClient,
-    );
-    const expiringAppsReport = await runExpiringAppsCheck(currentDate, emailClient, user);
-    const closedRenewalsReport = await runCloseUnsubmittedRenewalsCheck(currentDate, user);
+    const firstExpiryNotificationReport = renewalEnabled
+      ? await firstExpiryNotificationCheck(currentDate, emailClient)
+      : getReportToBeImplemented('FIRST EXPIRY NOTIFICATIONS');
```

Comment:
report to be implemented is probably wrong for this, lets write something like `getReportFeatureDisabled`